add pydata blog article and picture (#239)
SpareCores/sc-www: src/assets/articles/pydata_blog.md

---
# ~50 chars

> title: My personal thoughts and feelings about PyData London 2026

Reviewer: title-case + char limit constraints:

```suggestion
title: A Personal Recap of PyData London 2026
```

Or closer to the original:

```
My Personal Experience at PyData London 2026
```

date: 2026-06-09
# ~100 character

> teaser: "Personal takeaways from PyData London 2026: talks, MCP, AI burnout"

Reviewer: We need to keep the character limits, so maybe something like:

```suggestion
teaser: "From green energy maps to MCP and beating AI agent burnout: my personal recap of PyData London 2026."
```

# 320x220
image: /assets/images/blog/thumbnails/pydata_blog.webp

> image_alt: Gergely Daróczi and Tamás Hidvégi on PyData London 2026

Reviewer: ```suggestion
image_alt: Gergely Daróczi and Tamás Hidvégi at PyData London 2026
```

> author: Alex Trickey

Reviewer: this field is the blog article author, so should be you :)
you can credit Alex in the `image_alt` if you want

tags: [conference, talk]
---

> My personal thoughts and feelings about [PyData London 2026](https://pydata.org/london2026)

Reviewer: I don't think we need to repeat the title here, that will be on the top of the page anyway

> First of all, thanks a lot [@Gergely Daróczi](https://www.linkedin.com/in/daroczig/) for letting me join the [Spare Cores](https://www.linkedin.com/company/sparecores/) team at the conference (thx for the talk *SELECT instance FROM cloud WHERE workload = ? ORDER BY cost_efficiency* too, the topic was very close to me :)!) It was an exciting opportunity to attend such a great event. I gathered a lot of ideas, cloned a lot of repos, and as an extra I learned couple of things about myself, which I don’t think is a common experience from an IT conference :).

Reviewer: it was an invitation :)

```suggestion
First of all, thanks a lot [@Gergely Daróczi](https://www.linkedin.com/in/daroczig/) for inviting me to join the [Spare Cores](https://www.linkedin.com/company/sparecores/) team at the conference (thx for the talk `SELECT instance FROM cloud WHERE workload = ? ORDER BY cost_efficiency` too, the topic was very close to me :)!) It was an exciting opportunity to attend such a great event. I gathered a lot of ideas, cloned a lot of repos, and as an extra I learned couple of things about myself, which I don’t think is a common experience from an IT conference :).
```
